.github/workflows/rust.yml

```diff
@@ -85,11 +88,8 @@ jobs:
       uses: ./.github/actions/setup-builder
       with:
         rust-version: ${{ matrix.rust }}
+    - uses: Swatinem/rust-cache@9d47c6ad4b02e050fd481d890b2ea34778fd09d6 # v2.7.8
```

Comment:
I checked that we use the same action in DataFusion so this sounds like a good idea to me 👍 


https://github.com/apache/datafusion/blob/b41acf3c43ff259a22f9ecc2abdb17db58297fd8/.github/actions/setup-macos-aarch64-builder/action.yaml#L47

Thanks @assignUser and @iffyio